build/package/Installer.MSI.targets

```diff
@@ -45,8 +45,7 @@
       <GenerateMsiVersion CommitCount="$(CommitCount)"
                           VersionMajor="$(VersionMajor)"
                           VersionMinor="$(VersionMinor)"
-                          VersionPatch="$(VersionPatch)"
-                          ReleaseSuffix="$(ReleaseSuffix)">
```

Comment:
We weren't using ReleaseSuffix in 1.0.0-preview2.1 either:

https://github.com/dotnet/cli/blob/rel/1.0.0-preview2.1/build_projects/shared-build-targets-utils/Utils/Version.cs#L14-L37